apps/volk_option_helpers.h

```diff
@@ -6,6 +6,8 @@
 #define VOLK_VOLK_OPTION_HELPERS_H
 
 #include <string>
+#include <cstring>
```

Comment:
I'm personally a big fan of ICWY (Include what you use), where you use it, so in my world, this would be included in `volk_option_helpers.cc` itself, not in its symbol- and type-defining header.